### PR TITLE
Bump up kuroko2

### PR DESCRIPTION
**Pull request by riseshia**
It's time to bump up!!

I listed up what was changed.
And updating minor version sounds reasonable to me, how do you think? @cookpad/dev-infra

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## lastest(master branch)
+
+## v0.5.0
+
+- Use MutationObserver to observe log for work instance_linker.js correctly [#68](https://github.com/cookpad/kuroko2/pull/68)
+- Support multiple hosted domains [#106](https://github.com/cookpad/kuroko2/pull/106)
+- Let all job instance be target of cleanup_old_instance script [#107](https://github.com/cookpad/kuroko2/pull/107), [#108](https://github.com/cookpad/kuroko2/pull/108), [#109](https://github.com/cookpad/kuroko2/pull/109)
+- Support script revision [#110](https://github.com/cookpad/kuroko2/pull/110)
+- Upgrade from factory_girl to factory_bot [#111](https://github.com/cookpad/kuroko2/pull/111)
```
Comment (eagletmt) [inline CHANGELOG.md]: This change should not be included because it doesn't have any effect on kuroko2 users.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+- Don't use Rails 5.2 temporarily [#112](https://github.com/cookpad/kuroko2/pull/112)
+- Remove cmake from dependency [#113](https://github.com/cookpad/kuroko2/pull/113)
+- Add Confirm process when trying force-cancel [#114](https://github.com/cookpad/kuroko2/pull/114)
+- Use puma for feature specs [#115](https://github.com/cookpad/kuroko2/pull/115)
```
Comment (eagletmt) [inline CHANGELOG.md]: ditto

File: CHANGELOG.md (hunk 1 above)
Comment (riseshia) [reply CHANGELOG.md]: 4fe8b46 deleted

File: CHANGELOG.md (hunk 2 above)
Comment (riseshia) [reply CHANGELOG.md]: 4fe8b46 deleted

**Pull request merged**